packages/components/addon/components/hds/form/masked-input/base.hbs

```diff
@@ -21,14 +21,11 @@
       ...attributes
     />
   {{/if}}
-  <button
-    class="hds-form-masked-input__toggle-button"
-    type="button"
-    aria-label={{this.ariaLabel}}
+  <Hds::VisibilityToggle
```

Comment:
In figma we have this written as `_toggleVisability` just wanted to make sure there's alignment in naming.

Reply:
To me, "toggleVisability" sounds more like an action vs. an element which is used for performing the action.

Reply:
@andgen404 It looks like what @didoo is proposing is what we discussed today in our 1:1. That approach makes sense, so can you please move it to the Form Primitives page in Figma?